src/openforms/contrib/zgw/service.py

```diff
@@ -107,8 +110,10 @@ def create_report_document(
     submission_report.content.seek(0)
     base64_body = b64encode(submission_report.content.read()).decode()
 
+    assert options["auteur"], "auteur must be a non-empty string"
```

Comment:
I would expect this assertion in the `create_document` function, not here.

And then in the `document_options` it should be:

```
"auteur": options.get("auteur") or "Aanvrager",
```
to provide a fallback for empty or missing values (this requires a test to demonstrate empty auteur is handled! this test  is currently absent because the build would be failing due to the assert above)